app.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+var todoList = [
+  {task: "Walk the dog"},
+  {task: "Take out the trash"},
+  {task: "Buy groceries"},
+  {task: "Call mom"},
+  {task: "Clean the house"}
+];
+
+(function(){
+  angular
+    .module("todo", [])
+    .controller("todo_list", ToDoList);
```

Comment:
Just for convention purposes, I might write camelCase for JS here writing `"todoList"` instead of `"todo_list"`